07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,36 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
 
+<body>
+    <header>
+        <h1>Professional Portal</h1>
+        <nav>
+            <p>
+                Home Companies Positions
+            </p>
+        </nav>
+    </header>
+    <section>
+        <div class="green">
+            <h3>Hire an Employee</h3>
+            <p>
+                Lucas ipsum dolor sit amet organa dooku hutt chewbacca han organa watto darth darth. Kamino solo amidala windu. Jawa solo bespin moff
+            </p>
+            <button type="company">Sign up as company</button>
+        </div>
+        <div class="black">
+            <h3>Looking for a job</h3>
+            <p>
+                Lucas ipsum dolor sit amet organa dooku hutt chewbacca han organa watto darth darth. Kamino solo amidala windu. Jawa solo bespin moff
+            </p>
+            <button type="employee">Sign up as employee</button>
```

Comment:
i am not sure that `type=employee"` it's correct. 
Look [here](http://www.w3schools.com/tags/att_button_type.asp) to see what `type`s are available